pkg/controllers/nodeclaim/lifecycle/liveness_test.go

```diff
@@ -19,6 +19,9 @@ package lifecycle_test
 import (
 	"time"
 
+	"github.com/awslabs/operatorpkg/status"
+
+	operatorpkg "github.com/awslabs/operatorpkg/test/expectations"
```

Comment:
```suggestion
	. "github.com/awslabs/operatorpkg/test/expectations"
```

Can you dot-import this?

Reply:
We already have this - `. "sigs.k8s.io/karpenter/pkg/test/expectations"`

Reply:
Discussed offline.